runtime/ops/process.rs

```diff
@@ -227,6 +227,20 @@ fn create_command(
   command.args(args.args);
 
   if let Some(cwd) = args.cwd {
+    if let Ok(cwd_metadata) = std::fs::metadata(cwd.clone()) {
+      if !cwd_metadata.is_dir() {
+        return Err::<std::process::Command, _>(AnyError::msg(format!(
+          "The runtime path '{}' is not a directory.",
+          cwd.clone()
+        )));
+      }
+    } else {
+      return Err::<std::process::Command, _>(AnyError::msg(format!(
+        "Unable to access the runtime path '{}'.",
+        cwd.clone()
+      )));
+    }
```

Comment:
This will increase the amount of time it takes to launch a command. Instead, perhaps we should check if the cwd exists on a `NotFound` error?

Reply:
but wouldn't that be ambiguous regarding whether it's the command or the directory NotFound ?

Reply:
We could just always surface the cwd error when the cwd doesn't exist on a `NotFound` error. It would be almost essentially the same as this code.